deployments/external/docker-compose.yml

```diff
@@ -64,21 +64,23 @@ services:
     ports:
     - 8080:8080
     environment:
-      KC_HOSTNAME_URL: 'http://login.example.com'
-      KC_HOSTNAME_ADMIN_URL: 'http://login.example.com'
-      KC_HOSTNAME_STRICT_HTTPS: 'false'
+      KC_HOSTNAME: 'http://login.example.com'
+      KC_HOSTNAME_STRICT: 'false'
       KC_HTTP_ENABLED: 'true'
       KC_HTTPS_ENABLED: 'false'
       KEYCLOAK_ADMIN: admin
       KEYCLOAK_ADMIN_PASSWORD: Password1
-      KEYCLOAK_LOGLEVEL: DEBUG
+      KC_LOG_LEVEL: DEBUG
       KC_DB: postgres
       KC_DB_URL_HOST: keycloak-dbserver
       KC_DB_URL_DATABASE: keycloak
       KC_DB_USERNAME: postgres
       KC_DB_PASSWORD: Password1
+    volumes:
```

Comment:
This change is not required and we can drop it if we don't like it. Instead of including the dump of the whole DB I used the export/import realm feature of Keycloak. The configuration and data are now in JSON files. This solution should be resilient to any schema changes in Keycloak.